helpers/database.py

```diff
@@ -13,9 +14,22 @@ def __init__(self):
         }
         self.url = 'https://api.airtable.com/v0/appiGonw4Dnm3zSD1/Users'
 
+    def cache(self):
+        print('caching...')
+        data = requests.get(self.url, headers=self.headers).json()
```

Comment:
This will be able to fetch no more than 500 entries. If there are more, it will fail with http 429.